### PR TITLE
refactor: revamp homepage

### PR DESCRIPTION
**Pull request by lukicenturi**

### CODE REVIEW

File: components/home/features/details/EthProtocols.vue [hunk 1]
```diff
@@ -0,0 +1,70 @@
+<script setup lang="ts">
+const { t } = useI18n();
+
+const data = [
```
Comment (tewshi) [inline components/home/features/details/EthProtocols.vue:4]: think we'll need computed here, because of the translations

**Comment by lukicenturi** [comment]
@tewshi the plans section also updated

**Comment by kelsos** [comment]
@lukicenturi @tewshi don't merge it before I also have a look.

File: pages/index.vue [hunk 2]
```diff
@@ -1,5 +1,6 @@
 <script setup lang="ts">
 import { commonAttrs, getMetadata } from '~/utils/metadata';
+import HomeBanner from '~/components/home/HomeBanner.vue';
```
Comment (kelsos) [inline pages/index.vue:3]: we can probably remove this right?

File: components/home/features/details/SupportedExchangeDetails.vue [hunk 3]
```diff
@@ -1,4 +1,5 @@
 <script setup lang="ts">
+const { t } = useI18n();
 const exchanges = [
```
Comment (kelsos) [inline components/home/features/details/SupportedExchangeDetails.vue:3]: Can you check with the app and make sure we have the list right?

**Comment by kelsos** [comment]
So we need to make sure the list of exchanges is up to date with the app

Bittrex was the one that shutdown, I think bitmex is still operational, but let's do a cross-check. We might also have to somehow update the exchanges.png image because it has at least bittrext that is not supported in it.

Other than that all looks great :) thanks

**Comment by LefterisJP** [comment]
> think bitmex is still operational, but let's do a cross-check. 

bitmex is fine

**Comment by lukicenturi** [comment]
the list of exchanges is adjusted

**Review by kelsos — APPROVED** [review]
lgtm @lukicenturi thanks :)

**Pull request merged**